File: 프로그래머스/1/135808. 과일 장수/과일 장수.py
def solution(k, m, score):
    answer = 0
    
    score = sorted(score, reverse = True)
    for i in range(0, len(score), m):
        tmp = score[i : i + m]
        if len(tmp) == m:    
            answer += min(score[i : i + m]) * m
    
    return answer

File: 프로그래머스/1/135808. 과일 장수/과일 장수.js
function solution(k, m, score) {
    let answer = 0;
    score = score.sort((a, b) => a - b);
    while (score.length >= m) {
        const items = [];
        for (let i = 0; i < m; i++) {
            items.push(score.pop());
        }
        answer += Math.min(...items) * items.length;
    }
    return answer;
}